cocotb/__init__.py

```diff
@@ -106,8 +107,36 @@ def _reopen_stream_with_buffering(stream_name):
 plusargs = {}
 """A dictionary of "plusargs" handed to the simulation."""
 
-# To save typing provide an alias to scheduler.add
-fork = scheduler.add
+
+def current_test() -> cocotb.decorators.RunningTest:
+    """Returns a reference to the currently running test"""
+    global scheduler
```

Comment:
```suggestion
```

`global x` is for changing the meaning of `x = ...`. You don't assign to `scheduler` here, so `global` isn't doing anything other than making me look really hard for a `scheduler = ...` line that doesn't exist. Same below.